builtin/mainmenu/tab_settings.lua

```diff
@@ -247,7 +218,6 @@ local function formspec(tabview, name, tabdata)
 		"dropdown[0.25,3.2;3.3;dd_leaves_style;" .. leaves_style[1][1] .. ";"
 				.. getLeavesStyleSettingIndex() .. "]" ..
 		"box[3.75,0;3.75,3.45;#999999]" ..
-		"box[3.75,0;3.75,4.7;#999999]" ..
```

Comment:
The box is not entirely duplicate, its higher than the other one. Perhaps remove the other, smaller one?